custom_components/ramses_cc/climate.py

```diff
@@ -425,22 +408,25 @@ def svc_put_zone_temp(
         self._device.sensor._make_fake()
         self._device.sensor.temperature = temperature
         self._device._get_temp()
-        self.update_ha_state()
+        self.async_write_ha_state()
```

Comment:
So, `async_write_ha_state_delayed()` after a read, and `async_write_ha_state()` after a write?

Note that the new version of **ramses_rf** can invoke async API calls (have a look at **src/ramses_tx/command.py** for these APIs), and wait for the result (i.e. send a RQ & wait for the RP, send an W & wait for the I).

Before (0.2x.x), these calls were simply cast.

Reply:
Yes leave to wait - that reasoning isn't completely sound.

Invoking a command on a faked remote (which, BTW is technically stateless - other faked devices may not be, e.g. actuator) will cause the USB dongle to cast a packet on its behalf.

That `RQ`, `W` or `I` will 

Of course, this isn't a faked device, it's a zone, but teh same applies in a way.

With the new transport, which has QoS, you will be able to update without a delay. The QoS code is there, but is disabled by default.

If you want to play with it:
```yaml
ramses_cc:
  ramses_rf:
    disable_qos: false
```

Reply:
Yep - and async is best!